plugins/home/src/homePageComponents/Toolkit/Context.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { createContext } from 'react';
+import { Tool } from './Content';
+
+type ToolkitContextValue = {
+  tools: Tool[];
+};
+
+const Context = createContext<ToolkitContextValue>({ tools: [] });
+
+export const ContextProvider = ({
+  children,
+  tools,
+}: {
+  children: JSX.Element;
+  tools: Tool[];
+}) => {
+  const value: ToolkitContextValue = {
+    tools,
+  };
+
+  return <Context.Provider value={value}>{children}</Context.Provider>;
+};
+
+export const useTools = () => {
+  const value = React.useContext(Context);
+
+  if (value === undefined) {
+    throw new Error('useTools must be used within a Toolkit');
```

Comment:
```suggestion
    throw new Error('useTools must be used within a ContextProvider');
```

perhaps? 🤔  maybe i missed something